### ISSUE TITLE
installation error: about vc++10.0

### ISSUE THREAD

**Issue by genliu777** [larsmans/seqlearn#17]
hi, i do not know it is proper put here my puzzle  or not! when i install seqlearn, it shows error: "Microsoft visual c++ 10.0 is required(unable to find vcvarsall.bat)". but i install vc++2013 and vc++2008  on my laptop! i have searched a lot of forums but i have no idea what they say about it! so how to solve it!!?? 
 thanks!!!

**Comment by soupault** [comment]
@larsmans I agree. I can suggest to use AppVeyor as maintainer of `hmmlearn` does, so to have always ready-to-use code and also prepared versions for Windows: https://github.com/hmmlearn/hmmlearn/issues/30 .
